data/json/recipes/armor/legs.json

```diff
@@ -439,6 +439,7 @@
   {
     "result": "shorts_cargo",
     "type": "recipe",
+    "id_suffix": "from_fabric",
```

Comment:
The version from rags should be the default (no suffix). The one using scissors to cut off the bottom should be an alternative (with a suffix)